src/handlers/dhcp/DhcpStreamHandler.h

```diff
@@ -29,25 +31,29 @@ class DhcpMetricsBucket final : public visor::AbstractMetricsBucket
         Counter OFFER;
         Counter REQUEST;
         Counter ACK;
+        Counter total;
         Counter filtered;
 
         counters()
-            : DISCOVER("dhcp", {"wire_packets", "discover"}, "Total DHCP packets with message type DISCOVER")
-            , OFFER("dhcp", {"wire_packets", "offer"}, "Total DHCP packets with message type OFFER")
-            , REQUEST("dhcp", {"wire_packets", "request"}, "Total DHCP packets with message type REQUEST")
-            , ACK("dhcp", {"wire_packets", "ack"}, "Total DHCP packets with message type ACK")
-            , filtered("dhcp", {"wire_packets", "filtered"}, "Total DHCP wire packets seen that did not match the configured filter(s) (if any)")
+            : DISCOVER(DHCP_SCHEMA, {"wire_packets", "discover"}, "Total DHCP packets with message type DISCOVER")
+            , OFFER(DHCP_SCHEMA, {"wire_packets", "offer"}, "Total DHCP packets with message type OFFER")
+            , REQUEST(DHCP_SCHEMA, {"wire_packets", "request"}, "Total DHCP packets with message type REQUEST")
+            , ACK(DHCP_SCHEMA, {"wire_packets", "ack"}, "Total DHCP packets with message type ACK")
+            , total(DHCP_SCHEMA, {"wire_packets", "total"}, "Total DHCP wire packets seen that match the configured filter(s)")
```

Comment:
Sorry... I missed this one: "Total DHCP wire packets matching the configured filter(s)"

Reply:
done